pernosco-submit

```diff
@@ -126,12 +139,28 @@ while len(sys.argv) > 2:
     elif sys.argv[2] == "-x":
         del sys.argv[2]
         echo_commands = True
+    elif sys.argv[2].startswith("--substitute"):
+        if not rr_version_checked:
+            v = get_rr_version()
+            if v[0] <= 5 and v[1] < 4:
+                print("--substitute only works with rr version 5.4 or newer; found rr version %s.%s.%s" % v, file=sys.stderr)
+                sys.exit(1)
```

Comment:
I would rather just detect the failure via an rr error if rr doesn't support `--substitute`.

Reply:
Fair enough.